content/en/real_user_monitoring/application_monitoring/react_native/error_tracking.md

```diff
@@ -33,475 +33,13 @@ Your crash reports appear in [**Error Tracking**][1].
-}
-
-datadog {
-    checkProjectDependencies = "none" // this is needed in any case for React Native projects
-}
-```
-
-For the upload to work, you need to provide your Datadog API key. You can specify it as a `DATADOG_API_KEY` environment variable, or create a `datadog-ci.json` file at the root of your project containing the API key:
-
-```json
-{
-    "apiKey": "<YOUR_DATADOG_API_KEY>"
-}
-```
-
-You can also specify the Datadog site (such as `datadoghq.eu`) as a `DATADOG_SITE` environment variable, or as a `datadogSite` key in your `datadog-ci.json` file.
-For more information, see the [Datadog Android SDK Gradle Plugin][6].
-
-To run the plugin after a build run `(cd android && ./gradlew app:uploadMappingRelease)`.
-
-{{% /collapse-content %}}
-
-{{% collapse-content title="Automate the upload on each build" level="h5" %}}
-
-Install the plugin like in the previous step.
-
-Find the loop on `applicationVariants` in the `android/app/build.gradle` file. It should look like `applicationVariants.all { variant ->`.
-
-Inside the loop, add the following snippet:
-
-```groovy
-        if (project.tasks.findByName("minify${variant.name.capitalize()}WithR8")) {
-            tasks["minify${variant.name.capitalize()}WithR8"].finalizedBy { tasks["uploadMapping${variant.name.capitalize()}"] }
-        }
-```
-
-**Note**: Re-uploading a source map does not override the existing one if the version has not changed.
-
-{{% /collapse-content %}}
+If you have not set up the React Native SDK yet, follow the [in-app setup instructions][2] or see [React Native Crash Reporting and Error Tracking][3]. After setting up React Native, [follow these steps to enable React Native Crash Reporting and Error Tracking][101].
 
 ## Further reading
 
 {{< partial name="whats-next/whats-next.html" >}}
 
 [1]: https://app.datadoghq.com/rum/error-tracking
 [2]: https://app.datadoghq.com/rum/application/create
-[3]: /real_user_monitoring/application_monitoring/react_native/
-[4]: /real_user_monitoring/application_monitoring/ios/error_tracking/?tabs=cocoapods#symbolicate-crash-reports
-[5]: https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional
-[6]: https://github.com/DataDog/dd-sdk-android-gradle-plugin
-[7]: https://github.com/cwhenderson20/react-native-crash-tester
-[9]: https://fastlane.tools/
-[10]: https://appcenter.ms/
-[11]: https://www.bitrise.io/
-[12]: https://github.com/DataDog/datadog-ci/tree/master/packages/datadog-ci/src/commands/react-native#xcode
-[13]: https://github.com/DataDog/datadog-react-native-wizard
-[14]: https://github.com/DataDog/react-native-performance-limiter
-[15]: https://plugins.gradle.org/plugin/com.datadoghq.dd-sdk-android-gradle-plugin
-[16]: https://app.datadoghq.com/source-code/setup/rum
-[17]: https://github.com/DataDog/datadog-ci/blob/master/packages/datadog-ci/src/commands/react-native/README.md#inject-debug-id
+[3]: /real_user_monitoring/application_monitoring/react_native/error_tracking/
```

Comment:
This link opens the page the user is already on. Could you confirm if this link is expected to open a different page?